app/controllers/users_controller.rb

```diff
@@ -13,15 +13,16 @@ def create
   	end
   end
 
+  # Refactor this.
   def index
   	@users = User.all
     @userStats = User.stats
     @users = @users.sort_by do |user|
-      unless @userStats[user.id].nil?
-        @userStats[user.id][1]
-      else
+      if @userStats[user.id].nil?
         -1
-      end        
+      else
+        @userStats[user.id][1]
+      end
     end
```

Comment:
WTF is this method even doing?..... Let's rework how we query for these analytics.